docs/src/rsh/step/index.md

```diff
@@ -464,6 +464,22 @@ closeTo( Who, after, nonNetPayAmt )
 ```
 
 @{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the DApp after executing the function `{!rsh} after` in a step.
+
+As an example, codifying the prose above would look like:
+
+```reach
+// Has participant Who make a publication
+  Who.publish();
+// transfer the balance() and the non-network pay amount to Who
+  transfer([ balance(), ...nonNetPayAmt ]).to(Who);
+  commit();
+// end the DApp after executing the function after in a step
+  after();
+  exit();
+```
+
+By comparing this example to `closeTo`, you can see that using `closeTo` helps to save additional lines of code.
```

Comment:
You should use `{!rsh} closeTo`. And it's not just about saving typing, it's about saving thinking.